config/database.yml.sqlite

```diff
@@ -0,0 +1,11 @@
+development:
```

Comment:
Hi, this is looking great, but would you mind removing the this file from your PR, as it's unrelated to your changes?

https://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git

Thanks!